### DISCUSSION TITLE
Dados da Transação PIX

### DISCUSSION THREAD

**Discussion by otavioz** [andreroggeri/pynubank#363] [Q&A] [answered]
Pelo que entendi a função _get_bank_statement_ retorna as transações realizada na conta incluindo PIX enviados/recebidos,  porém ao executá-la não encontrei nenhuma de minhas transações nos registros. Contudo vejo no código o suporte para o tipo de transação PIX, o retorno dessas informações não está implementado ou eu que estou "comendo mosca"?

**Comment by andreroggeri** [comment, marked as answer]
A API do Nubank está em mudança e o endpoint antigo está começando a retornar menos informações ([Mais info aqui](https://github.com/andreroggeri/pynubank/issues/341)).

Você poderia testar com o `get_account_statements_paginated` e confirmar se está retornando essas transações?

**Reply by otavioz** [reply]
André, muito obrigado pelo retorno; `get_account_statements_paginated` está retornando as transações. Obrigado!!
